packages/core-mobile/app/services/fcm/AppCheckService.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  firebase,
+  FirebaseAppCheckTypes
+} from '@react-native-firebase/app-check'
+import Logger from 'utils/Logger'
+
+class AppCheckService {
+  init = (): void => {
+    const rnfbProvider = firebase
+      .appCheck()
+      .newReactNativeFirebaseAppCheckProvider()
+    rnfbProvider.configure({
+      android: {
+        provider: __DEV__ ? 'debug' : 'playIntegrity',
+        debugToken:
+          'some token you have configured for your project firebase web console'
+      },
+      apple: {
+        provider: __DEV__ ? 'debug' : 'appAttestWithDeviceCheckFallback',
+        debugToken: process.env.APPCHECK_DEBUG_TOKEN_APPLE
+      }
+    })
+
+    firebase
+      .appCheck()
+      .initializeAppCheck({
+        provider: rnfbProvider,
+        isTokenAutoRefreshEnabled: true
+      })
+      .catch(reason => {
+        Logger.error(`initializeAppCheck failed: ${reason}`)
+      })
+  }
+
+  verifyAppCheck = async (): Promise<void> => {
+    try {
+      const { token } = await firebase.appCheck().getToken(true)
```

Comment:
```suggestion
      const { token } = await this.getToken()
```

Reply:
nice catch!
[Add comment for unused verifyAppCheck](https://github.com/ava-labs/core-mobile/pull/1659/commits/e42f2a7031b3fa91890b0b28cf7748e58fd851d5)